Gds/src/fprime_gds/flask/static/index.html

```diff
@@ -292,7 +292,7 @@ <h2>File Uplink</h2>
                             <input class="col-2" type="text" v-model="destination" pattern="/.*" />
                             <div class="col-4">
                                 <label>Add Files to Uplink:</label>
-                                <input type="file" class="btn btn-active" v-bind:files="selected" multiple v-on:change="handelFiles" />
+                                <input type="file" class="btn btn-active" v-bind:files="selected" multiple v-on:change="handleFiles" />
```

Comment:
?

Reply:
The other usage of that API change below.